packages/base/catalog-entry.gts

```diff
@@ -44,6 +44,28 @@ export class CatalogEntry extends CardDef {
       return primitive in card || isEqual(baseCardRef, this.ref);
     },
   });
+  // If it's not a field, then it's a card
+  @field isField = contains(BooleanField, {
+    computeVia: async function (this: CatalogEntry) {
+      let loader = Loader.getLoaderFor(Object.getPrototypeOf(this).constructor);
+
+      if (!loader) {
+        throw new Error(
+          'Could not find a loader for this instance’s class’s module',
+        );
+      }
+
+      let card: typeof BaseDef | undefined = await loadCard(this.ref, {
+        loader,
+        relativeTo: this[relativeTo],
+      });
+      if (!card) {
+        throw new Error(`Could not load card '${this.ref.name}'`);
+      }
+
+      return 'isFieldDef' in card && card.isFieldDef;
```

Comment:
There is a function added in last PR called `isFieldDef` in runtime-common

Reply:
Great, will use it